test/runtests.jl

```diff
@@ -13,6 +14,8 @@ using Cbc
 using OSQP
 using TimeSeries
 using ParameterJuMP
+using Compat: Test
```

Comment:
Why do we need to use compat here to import the Test package?

Reply:
Per the TestSetExtensions docs:  https://github.com/ssfrr/TestSetExtensions.jl

Reply:
I don't think that's necessary, the README hasn't been updated since 2017

Reply:
You are correct.  Thanks.